### DISCUSSION TITLE
Estructura y convenciones de canales en Slack

### DISCUSSION THREAD

**Discussion by murillodigital** [CRDevOpsCloudNative/comunidad#2] [General]
# Definición inicial de estructura, convenciones y responsabilidades en Slack

Slack es el medio principal de comunicación de la Comunidad DevOps + Cloud Native de Costa Rica. Con el fin de lograr comunicación ordenada, respetuosa y enriquecedora, es importante definir algunos criterios básicos de estructura y funcionamiento de la herramienta.

El objetivo de esta discusión es definir:

- La convención a utilizar para nombrar los diferentes canales
- El listado inicial de canales
- Definir las reglas de conducta básicas y en los canales que aplique, la asignación de moderadores

# Convención para nombres de canales

Se definen los siguientes prefijos para determinar la naturaleza de los canales:

| Prefijo | Descripción |
| --------- | ----------- |
| evento- | Canales para coordinación de evento y efímeros para interacción alrededor de eventos específicos |
| sig- | Special Interest Groups: Espacio para la conversación alrededor de tecnologias, practicas o areas conocimiento especifico |
| comunidad- | Canales asociados a iniciativas de la comunidad, coordinación y oportunidades

## Canales para eventos

Los canales asociados a eventos podrán ser persistentes, para anunciar y discutir sobre los distintos eventos que están programados, o efímeros asociados a eventos específicos.

### Canales persistentes

| Nombre de canal | Objetivo |
| ----------------- | --------- |
| evento-workshops | Canal para el aviso y discusión abierta sobre el track de workshops de la comunidad |
| evento-charlas | Canal para el aviso y discusión abierta sobre el track de charlas de la comunidad |
| evento-externo | Espacio para compartir eventos en cualquier parte del mundo con nuestra comunidad  y para que otras comunidades en Costa Rica puedan compartir sus iniciativas |

## Special Interest Groups

Grupos de interés en áreas específicas

| Nombre de canal | Objetivo |
| ----------------- | --------- |
| sig-kubernetes | Temas generales asociados a Kubernetes en sus diferentes distribuciones, helm, kustomize y herramientas similares |
| sig-devops | Discusión abierta sobre todos los aspectos técnicos, culturales y organizacionales de DevOps |
| sig-dataops | Discusión abierta sobre todo lo relacionado con DataOps - herramientas, cultura, procesos y más |
| sig-devsecops | Discusión abierta sobre ciber seguridad en contextos nativos para la nube, su integración con la cultura DevOps, herramientas y más |
| sig-iac | Discusión abierta sobre infrastructure as code e infrastructure as software |
| sig-finops | Discusión abierta sobre FinOps, gestión de costos en la nube, vibisibilidad, etc |
| sig-observability | Discusión abierta sobre observabilidad, monitoreo, etc |
| sig-arquitectura | Discusión abierta sobre arquitectura de sistemas e infraestructura nativa para la nube |
| sig-mlops | Discusión abierta sobre MLOps y Machine Learning en general |
| sig-chaos | Discusión sobre Chaos Engineering |

## Comunidad

| Nombre de canal | Objetivo |
| ----------------- | --------- |
| comunidad-empleos | Espacio para compartir oportunidades de empleo y desarrollo profesional |
| comunidad-foodops | Coordinación y discusión abierta sobre la iniciativa de acción social FoodOps |
| comunidad-educacion | Espacio para compartir oportunidades académicas con la comunidad |
| comunidad-intros | Espacio para que nos introduzcamos y contemos un poco sobre nosotros |
| comunidad-principantes | No estás seguro si la pregunta es muy básica o te sentís inhibido de hablar de temas básicos en otros canales? Este canal es donde hablamos sobre todos los temas sin importar que tan básicos sean |

# Guía y código de conducta

Al participar en las actividades y medios de comunicación de la Comunidad DevOps + Cloud Native de Costa Rica, incluyendo nuestro Slack Workspace, usted esta de acuerdo en seguir las siguientes reglas básicas.

La definición de términos como "acoso", "discriminatorio" y "ofensivo" podrá ser determinado a la discreción del equipo de organizadores de la comunidad.

## Una comunidad abierta a todos los niveles de experiencia

La comunidad promueve el intercambio de conocimiento y la integración de personas en cualquier nivel de experiencia y conocimiento.

## Usar lenguage apropiado

Sea amable con todos los miembros de la comunidad. Evite cualquier tipo de ataque personal, amenazas, insultos o lenguaje ofensivo. Si considera que se está dando comportamiento inapropiado en Slack o en otros canales de comunicación o interacción oficiales de la comunidad, por favor comuníquelo a los organizadores.

## Espacio libre de acoso

Ningún tipo de acoso será tolerado en la comunidad. Los miembros de la comunidad nos compremetemos por crear un ambiente amigable y seguro para todos, sin importar sus habilidades, identidad de género, raza, etnia, apariencia física, religión,  edad o cualquier otra característica.

Se espera que todos los miembros de la comunidad respeten diferencias de opinión.

Acoso incluye, pero no se limita a: uso de lenguaje abusivo, ofensivo o degradante, intimidación, acecho, acoso fotográfico o en grabaciones de video, contacto físico no apropiado, imágenes con contenido sexual, avances sexuales no bienvenidos o solicitud de favores sexuales.

## Evite mensajes repetitivos, no solicitados o comerciales

Respete los deseos de los otros miembros de la comunidad en relación a mensajes directos. Por favor sea conciente y respetuoso al "taggear" a otros en sus mensajes.

No utilice los canales de la comunidad para ofrecer productos o servicios, con excepción de aquellos canales creados con ese propósito.

## Evite actividades ilegales

No publique nada que pueda constituir una acción contraria a la ley. Por ejemplo: enlaces a contenido "pirateado" o cualquier otro material cubierto por términos de confidencialidad o de propiedad intelectual.

## Protección de datos de los miembros

La comunidad únicamente registrará y almacenará datos de naturaleza personal cuando exista un propósito directamente relacionado a la comunidad que lo justifique. Ninguna información de los miembros de la comunidad será compartida con otras organizaciones con intereses comerciales o de mercadeo, con excepción de casos en los que los miembros hayan dado su autorización explicita.

## Violaciones a este código de conducta

En caso de incurrir en la violacion de alguna de las normas de la comunidad, los administradores podran presentar sanciones que incluyen desde la expulsion de las redes sociales, Slack Workspace, GitHub Repo, e inclusive la negacion de participación en eventos.

**Comment by stvansolano** [comment]
@murillodigital que tal un canal permanente de aprendizaje por temas principales? Algo como #beginners donde gente devs o recien incorporados puedan ponerse al corriente y mejorar sus skills.

Pasa mucho que la gente con más tiempo se vuelve experta y otros van a diferente velocidad, hasta que se pierde el hilo o balance mientras se ponen al día o se aprende más.

**Reply by murillodigital** [reply]
Agregue un canal `comunidad-principiantes` decime que pensas?

**Comment by rcor** [comment]
"Ningún tipo de acoso será tolerado en la comunidad."
Se tiene que ser mas especifico sobre que se considera acoso, ademas se tiene que agregar como la comunidad va a a responder sobre una accion de acoso ( se le saca solo de  slack, no puede ir a eventos etc etc )

Agreagaria una seccion de proteccion de los datos de los miembros 

Tipo se prohibe distribuir los correos, fotografias, numeros de telefono de los miembros de la comunidad a terceros o utilizarlos con fines comerciales, publicitarios etc etc

**Reply by murillodigital** [reply]
@rcor super buena idea lo de uso de datos de miembros! Ya el tema de "que sucede en caso de violación a las normas" es la última sección del documento:

```Si viola cualquier porción de este código de conducta, su acceso a los recursos de la comunidad como el Slack Workspace o futuros eventos podría ser eliminado.```

Dejame ver que me encuentro en referencias a definición más clara de "acoso", si has visto algun buen ejemplo en otro comunidad compartímelo.

**Reply by murillodigital** [reply]
@rcor hice un intento de agregar una mejor definición de acoso (basandome en el código de conducta del linux foundation), dale una ojeada. De igual manera, y a sabiendas de que estos criterios pueden ser ambigüos, anote claramente que el equipo de organización también se encuentra en libertad de caracterizar comportamientos como acoso o abusivos según su discreción.

**Reply by rcor** [reply]
@murillodigital  se tiene que plantear un protocolo en caso de resolucion de conflictos y acoso, pero bueno eso es un laburo
Yo pondria en lugar de 

`Si viola cualquier porción de este código de conducta, su acceso a los recursos de la comunidad como el Slack Workspace o futuros eventos podría ser eliminado.`
En caso de incurrir en la violacion de alguna de las normas  de la comunidad, los administradores podran presentar alguna sancion que incluye desde la expulsion de las redes sociales hasta la negacion de participacion de eventos ( esto se puede revisar mejor cuales serian los castigos y que implica )

Algo bueno que se puede agregar es que cualquier expositor cede sus derechos sobre la exposicion y ejemplos utilizados para el uso de la comunidad y no confines comerciales. Un ejemplo pero seria bueno cuidarse ahi

**Comment by murillodigital** [comment]
He incorporado la retroalimentación de los miembros que colaboraron. Gracias a @rcor y @stvansolano por su participación. La discusión se da por concluída. Estaré transfieriendo estos resultados al repositorio.
